algebra/src/groups/group_vec.rs

```diff
@@ -47,10 +43,15 @@ impl<G: Group> GroupVec<G> {
     pub fn into_iter(&self) -> IntoIter<G> {
         self.0.clone().into_iter()
     }
+
+    pub fn rand<R: Rng + ?Sized>(len: u16, rng: &mut R) -> Self {
+        Self::new((0..len).map(|_| G::rand(rng)).collect::<Vec<G>>())
+    }
 }
 
 impl<G: Group> Distribution<GroupVec<G>> for Standard {
     #[inline]
+    // better use the specific function `rand` which allows to specify the length of the vector
```

Comment:
Please put this comment as argument of the unimplemented!

Reply:
Ok